ios/RecipesApp/AppDelegate.mm

```diff
@@ -10,7 +11,6 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   // You can add your custom initial props in the dictionary below.
   // They will be passed down to the ViewController used by React Native.
   self.initialProps = @{};
-
```

Comment:
do we need this?

Reply:
no, actually deleting that line is just a typo, I will add the space back